src/Storefront/Resources/app/storefront/src/plugin/main-menu/offcanvas-menu.plugin.js

```diff
@@ -93,30 +84,20 @@ export default class OffcanvasMenuPlugin extends Plugin {
      * @private
      */
     _getLinkEventHandler(event, link) {
-        if (!link) {
-            const initialContentElement = DomAccess.querySelector(document, this.options.initialContentSelector);
-            this._content = initialContentElement.innerHTML;
 
-            if (window.Feature.isActive('CACHE_REWORK')) {
```

Comment:
@mitelg I had to remove the feature flag check here, so it always works like that. I tested it with and without activated flag. Seems to work pretty smooth. Should be no issue from code perspective, because the method is private, but do you see any issues with your cache rework?

Reply:
if it works, it should be fine :blush: it was just about the initial loading. but as the whole plugin now works differently, it should be okay